__e2e__/init.test.ts

```diff
@@ -94,3 +94,32 @@ test('init --template file with custom directory', () => {
   let dirFiles = fs.readdirSync(path.join(DIR, customPath));
   expect(dirFiles).toEqual(customTemplateCopiedFiles);
 });
+
+test('init skips installation of dependencies with --skip-install', () => {
+  createCustomTemplateFiles();
+  let templatePath = path.resolve(DIR, 'custom', 'template');
+  if (process.platform === 'win32') {
+    templatePath = templatePath.split('\\').join('/');
+  }
+
+  const {stdout} = runCLI(DIR, [
+    'init',
+    '--template',
+    `file://${templatePath}`,
+    'TestInit',
+    '--skip-install',
+  ]);
+
+  expect(stdout).toContain('Run instructions');
+
+  // make sure we don't leave garbage
+  expect(fs.readdirSync(DIR)).toContain('custom');
+
+  let dirFiles = fs.readdirSync(path.join(DIR, 'TestInit'));
+
+  expect(dirFiles).toEqual(
+    customTemplateCopiedFiles.filter(
+      file => !['node_modules', 'yarn.lock'].includes(file),
```

Comment:
can we also check if those files exist in previous test "init --template file with custom directory"

Reply:
We already have it 
https://github.com/react-native-community/cli/blob/2c08a8254cacae74ae2cbdd4e25d8506d4983d15/__e2e__/init.test.ts#L95

Reply:
I assume you're looking forward to have another test case with the custom directory and `--skip-install` flag.

Reply:
Nah, I don't care too much about it

Reply:
I'm sorry, what do you expect more from my side in the context of this PR?

Reply:
You're right, now I noticed `customTemplateCopiedFiles` includes these :)